pkg/api/addons.go

```diff
@@ -680,6 +666,16 @@ func (cs *ContainerService) setAddonsConfig(isUpgrade bool) {
 		},
 	}
 
+	if !cs.Properties.IsAzureStackCloud() {
```

Comment:
Not on Azure Stack?

Reply:
Right. Because we don't have MCR images of cluster-proportional-autoscaler (that I know of), I think Azure Stack is more complicated.

cc @jadarsie

Reply:
Since we're in fact using MCR, this will be Azure Stack safe, so I'll remove this guard.